pytest_allure_dsl.py

```diff
@@ -45,13 +45,13 @@ def allure_dsl(request):
 @pytest.fixture('function', autouse=True)
 def __allure_dls_pre_post_actions__(request, allure_dsl):
     if request.config.option.allure_dsl:
-        MASTER_HELPER.description(allure_dsl.description)
+        allure_dsl.setup()
 
         try:
             yield
         finally:
             if request.config.option.allure_dsl:
```

Comment:
request.config.option.allure_dsl may be changed during execution?

Reply:
Done